airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -220,27 +219,14 @@ class PodManager(LoggingMixin):
     def __init__(
         self,
         kube_client: client.CoreV1Api = None,
```

Comment:
```suggestion
        kube_client: client.CoreV1Api,
```

Shouldn't this be required now?